packages/task/src/browser/task-service.ts

```diff
@@ -1105,6 +1105,7 @@ export class TaskService implements TaskConfigurationClient {
      * @param task The task to configure
      */
     async configure(token: number, task: TaskConfiguration): Promise<void> {
+        Object.assign(task, {label: this.taskNameResolver.resolve(task)});
```

Comment:
Minor: looks like here you have a formatting issue, could you check it please

Reply:
Fixed in a new commit !
Thanks Roman !